src/components/Send.css

```diff
@@ -25,6 +25,21 @@ input::-webkit-inner-spin-button {
   padding: 0.2rem 0.5rem;
 }
 
+.send.maker-running form,
+.send.maker-running .collaborators-selector,
+.send.maker-running button[type='submit'] {
+  filter: blur(2px);
+}
+
+/* figma design would like to have the values shown when a taker operation
+   is running - since we do not support this by now, lets blur the
+   forms in the meantime, just like when a maker is running. */
```

Comment:
```suggestion
```

I'd leave that out. Not really relevant to document here imo.

Reply:
Removed.